firefox_puppeteer/ui/toolbars.py

```diff
@@ -223,6 +223,10 @@ def notification_popup(self):
         """
         return self.marionette.find_element(By.ID, "notification-popup")
 
+    def open_identity_popup(self):
+        self.locationbar.identity_box.click()
```

Comment:
`self.locationbar` does not exist in the context of the locationbar class. It is simply `self`.